explorer/Explorer.tsx

```diff
@@ -589,13 +589,18 @@ export class Explorer
         // set given variable IDs as dimensions to make Grapher
         // download the data and metadata for these variables
         const dimensions = config.dimensions ?? []
-
-        yVariableIdsList.forEach((yVariableId) => {
-            dimensions.push({
+        const variablesToLoad = yVariableIdsList
+            // Filter out variableIds that are already present in the dimensions array
+            .filter(
+                (yVariableId) =>
+                    !dimensions.some((d) => d.variableId === yVariableId)
```

Comment:
```suggestion
                    !dimensions.some((d) => d.variableId === yVariableId && d.property === DimensionProperty.y)
```

Some scatters have the same variable as x and y dimensions, so maybe it's worth being more defensive and only filtering out y duplicates at this spot?